### DISCUSSION TITLE
Does not reply to the correct person

### DISCUSSION THREAD

**Discussion by matan869** [GorvGoyl/ChatGPT-Writer#83] [Report Bugs and Issues]
![image](https://user-images.githubusercontent.com/125085207/229214034-adf0ed6b-f323-48b8-a6df-f7501a931b8d.png)

Customer is named Janice (why not check sender name?), My name is John, and it's saying "Hello John". I've tried different ways to ask it to say Hi Janice but it never works.

Recommendations: Check Sender name and apply to the reply, and ask for the name of the person who's replying so it doesnt do [insert your name here] all the time.

**Comment by GorvGoyl** [comment]
use this prompt: 

reply to janice as john: Hi Janice, we are checking which piece is less than 1 gram and we will send you details.
